src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,76 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const countLinks = Math.ceil(total / perPage);
+  const links = getNumbers(1, countLinks);
+
+  return (
+    <ul className="pagination">
+      <li
+        // className="page-item disabled"
+        className={cn('page-item', {
+          disabled: currentPage === 1,
+        })}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1 ? 'true' : 'false'}
+          onClick={() => currentPage > 1 && onPageChange(currentPage - 1)}
```

Comment:
Using the `href` attribute with a hash (`#prev`) will modify the URL when clicked. This can lead to unexpected behavior in a single-page application. Consider using `button` elements for pagination controls or preventing the default action of the anchor tag in the `onClick` handler.